examples/input/keyboard_input.rs

```diff
@@ -22,4 +22,7 @@ fn keyboard_input_system(keyboard_input: Res<Input<Key>>) {
     if keyboard_input.just_released("a") {
         info!("'A' just released");
     }
+    if keyboard_input.pressed("A") {
+        info!("capital 'A' is being pressed");
+    }
```

Comment:
More info: https://gist.github.com/Vrixyz/16e7d7eb204f31f768c96f59f4bfa40d

TLDR: difficult to cache a Key::Character with modifiers.

I think we'll want our cache to rely on `key_without_modifiers` rather than `logical_key`, and aside from that our `ReceivedCharacter` event can take the `logical_key` because it doesn't rely on caching.

It needs an update from winit to expose the key without modifiers though.

Reply:
https://github.com/Vrixyz/bevy/pull/3 should fix it.